dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
@terrywbrady : thanks for the update. This may be a bit of a lost cause here...it's possible that a separate Test file is adding these unnamed groups. Let's just go with this code as-is, as it's not worth trying to find the needle in the haystack here.  But, let's add in a comment as to why the `if` statement is here. Something like:
```
// Ignore any unnamed groups. This is only necessary when running unit tests via a persistent database (e.g. Postgres) as unnamed groups may be created by other tests.
```